nilearn/_utils/testing.py

```diff
@@ -56,6 +60,34 @@ def assert_warns(warning_class, func, *args, **kw):
         return output
 
 
+def assert_warns_regex(warning_class, warning_regex,
```

Comment:
Nice catch.  There were two more commits from the `fix_warnings` branch that I missed here (from their commit comment, I couldn't tell).  Rebased to remove them.